index.json

```diff
@@ -232,5 +250,23 @@
     "path": "rocket-pool-04-2024",
     "sourceCodeUrl": "https://github.com/rocket-pool/rocketpool",
     "commit": "84ac19872dda7ca9c39c4f7349159d0e984130b9"
+  },
+  {
+    "name": "Propy",
+    "date": "05/2024",
+    "desc": "A review of the Propy NFT staking smart contracts update.",
+    "auditors": ["Tanya Bushenyova", "Oleksii Matiiasevych", "Anderson Lee"],
+    "path": "propy-nft-staking-v2-05-2024",
+    "sourceCodeUrl": "https://github.com/Propy/Propy.PRONFTStaking",
+    "commit": "7e5cac32f9451d5ac67d802562262a5bfbf249c9"
+  },
+  {
+    "name": "Propy",
+    "date": "05/2024",
+    "desc": "A review of the Propy Auction smart contracts update.",
```

Comment:
"Update" sounds like a diff audit. The report itself doesn't mention the update, it just says V2.